runtime/tests/runtime_integration_tests/test_disable.rs

```diff
@@ -30,4 +36,31 @@ fn test_disable_attestation_cmd() {
         resp_hdr.fips_status,
         MailboxRespHeader::FIPS_STATUS_APPROVED
     );
+
+    // test that attestation fails
+    let sign_cmd = SignCmd {
+        handle: ContextHandle::default(),
+        label: TEST_LABEL,
+        flags: SignFlags::empty(),
+        digest: TEST_DIGEST,
+    };
+    let resp = execute_dpe_cmd(&mut model, &mut Command::Sign(sign_cmd), DpeResult::Success);
+    let Some(Response::Sign(_)) = resp else {
+        panic!("Wrong response type!");
+    };
+
+    let certify_key_cmd = CertifyKeyCmd {
+        handle: ContextHandle::default(),
+        label: TEST_LABEL,
+        flags: CertifyKeyFlags::empty(),
+        format: CertifyKeyCmd::FORMAT_X509,
+    };
+    // the signature should be unable to be verified so expect a CFI error triggered
+    // by the ecc384 driver
```

Comment:
Hmm, that's unexpected. I would expect the driver to be able to verify the signature. The CDI used to derive the key won't be the same, so the RT pubkey won't match the key in the RT cert. But the key itself should be usable to sign stuff, right?

Reply:
It's because the RT alias public key which we get from the FHT no longer matches the zeroed private key, so verification fails in the driver itself.

Reply:
I am now updating the rt pub key whenever disable attestation is called so that the pub key and the priv key match.